src/ui/ContextSwitcher/ContextSwitcher.tsx

```diff
@@ -110,8 +127,30 @@ const Loader = ({ isLoading }) => {
   )
 }
 
-Loader.propTypes = {
-  isLoading: PropTypes.bool.isRequired,
+interface Context {
+  owner: {
+    avatarUrl: string
+    username: string | null
+  } | null
+  pageName: string
+}
+
+export interface Props {
```

Comment:
Can this interface be a bit more explicit as to what it's types are for if we're exporting it?